neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jModule.kt

```diff
@@ -32,6 +32,8 @@ class Neo4jModule : PrimeModule {
         if (System.getenv("ACCEPTANCE_TESTING") == "true") {
             RunnableKotlinScript(ClasspathResourceTextReader("/AcceptanceTestSetup.kts").readText()).eval<Any?>()
         }
+
+//        Neo4jStoreSingleton.subscribeToSimProfileStatusUpdates()
```

Comment:
If `subscribeToSimProfileStatusUpdates` is not required anymore, please remove the commented code

Reply:
Using it now :)